Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG NODE_VERSION=18.14.0
+
+FROM node:${NODE_VERSION}-alpine
+
+# Set the working directory to /cra-template-truemark inside the container
+WORKDIR /cra-template-truemark
+
+# copy package.json file content
+COPY package*.json ./
+
+# Copy app files
+COPY . .
+
+# Install project dependencies
+RUN npm install
+
+# Create a batch script to run both npm start,cypress and unit tests
+RUN echo "@echo off" > C:\start-and-test.bat \
```

Comment:
It should be possible to run the server, unit and e2e tests separately. You will have to research on this, using same docker for these all won't be fruitful as we want these commands to run off at a time. e.g. we want to run the server for doing the development work and use unit tests command only when we want to run unit tests after writing some tests

Reply:
You can take a new card for this purpose, let's create a new card for this and take in next sprint

Reply:
@coolprobn dai, [Here](https://trello.com/c/MOq1Bz2x/33-run-the-server-unit-and-e2e-tests-separately-in-docker) is the card created for above task.